controllers/favourite.controller.js

```diff
@@ -0,0 +1,46 @@
+const {
+  udpateFavMealPlanStatus,
+  getUserFavouriteMealPlans,
+  udpateFavRecipeStatus,
+  getUserFavouriteRecipes,
+} = require("../services/favourite.service");

```

Comment:
change the export and import type. Same as you do with controller in router file